### ISSUE TITLE
Pressing enter after block comment causes loss of indentation

### ISSUE THREAD

**Issue by bobef** [spadgos/sublime-jsdocs#391]
This is Sublime 3 3095. If you press enter after the block comment the indentation is lost and the cursor is at the start of the line. This behaviour started after installing this plugin. Confirmed in PHP and JS files.

**Comment by spadgos** [comment]
This is working for me (same result in JS and PHP mode)
![out](https://cloud.githubusercontent.com/assets/426222/10214087/1bd7693a-6815-11e5-8821-64a3b9b616bc.gif)

Is that what you're talking about?

**Comment by bobef** [comment]
Yes, it is the same what I mean but seems it works for you. I had this
experience on OSX and files with tab indent.
